tofhir-server/src/main/scala/io/tofhir/server/util/CsvUtil.scala

```diff
@@ -0,0 +1,182 @@
+package io.tofhir.server.util
+
+import akka.stream.scaladsl.{Concat, FileIO, Flow, Framing, Sink, Source}
+import akka.util.ByteString
+import com.opencsv.CSVParserBuilder
+import io.tofhir.engine.Execution.actorSystem
+import io.tofhir.engine.Execution.actorSystem.dispatcher
+import io.tofhir.server.common.model.InternalError
+
+import java.io.File
+import java.nio.file.StandardOpenOption
+import scala.concurrent.Future
+
+object CsvUtil {
+
+  /**
+   * Flow to remove carriage returns from a ByteString
+   */
+  private val removeCarriageReturns: Flow[ByteString, ByteString, _] = Flow[ByteString].map { line =>
```

Comment:
Can you please add the reasoning (why you needed this) to the comment of this function?